src/NHibernate.Test/Async/NHSpecificTest/NH1553/MsSQL/SnapshotIsolationUpdateConflictTest.cs

```diff
@@ -135,10 +135,10 @@ private void SavePerson(Person p)
 						        .And.Property("Identifier").EqualTo(p2.Id);
 
 					Assert.That(
-						() =>
+						async 						() =>
```

Comment:
After a quick look I saw that there are several strange formattings related to adding the `await/async` keywords like this one. Will try to fix this [issue](https://github.com/maca88/AsyncGenerator/issues/18).